website/integrations/services/fleet/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Integrate with Fleet
+sidebar_label: Fleet
+support_level: authentik
+tags:
+    - integration
+    - device-management
+authentik_enterprise: true
+authentik_preview: true
+---
+
+## What is Fleet
+
+> Fleet is an open source device management (MDM) platform for vulnerability reporting, detection engineering, device health monitoring, posture-based access control, managing unused software licenses, and more.
+>
+> -- [Fleet](https://fleetdm.com/)
+
+## Preparation
+
+By the end of this integration, your users will be able to log into Fleet using their authentik credentials.
+
+Your authentik and Fleet instance must both be running and accessible on an HTTPS domain.
+
+### Placeholders
+
+The following placeholders are used in this guide:
+
+- `authentik.company`: The FQDN of the authentik installation.
+- `fleet.company`: The FQDN of the Fleet installation.
+
+## authentik configuration
+
+The workflow to configure authentik as a single sign-on for Fleet involves creating an application and SAML provider pair. Following this configuration process will generate the necessary metadata you will use to configure Fleet to trust authentik as an identity provider.
+
+### 1. Create an application and provider
+
+1. From the authentik Admin interface, navigate to **Applications -> Applications** and click **Create with Provider** to create an application and provider pair.
+
+2. Enter "Fleet" for the **App name** and click **Next**.
```

Comment:
```suggestion
2. For the **App name** enter `Fleet` and click **Next**.
```

Reply:
Ideally, this section after Step 1 would be a bulleted list of the panels on the "wizard", (Applicatin, Choose a Provider, etc), with the required values after the colon. Focusing on the UI makes it easy for the user (doesn't have to read full sentences or lose their place in the UI) and for us, writing it. ;-)

Here's an example (Mautic) of that format:
<img width="926" alt="Screenshot 2025-03-21 at 3 10 51 PM" src="https://github.com/user-attachments/assets/e607814c-3825-481b-abf0-d983f81ac842" />

However... if you don't have time right now we can do that later.